doc/source/operating/snitch.rst

```diff
@@ -28,16 +28,18 @@ In cassandra, the snitch has two functions:
 
 Dynamic snitching
 ^^^^^^^^^^^^^^^^^
-
 The dynamic snitch monitor read latencies to avoid reading from hosts that have slowed down. The dynamic snitch is
 configured with the following properties on ``cassandra.yaml``:
 
 - ``dynamic_snitch``: whether the dynamic snitch should be enabled or disabled.
 - ``dynamic_snitch_update_interval_in_ms``: controls how often to perform the more expensive part of host score
   calculation.
-- ``dynamic_snitch_reset_interval_in_ms``: if set greater than zero, this will allow 'pinning' of replicas to hosts
-  in order to increase cache capacity.
-- ``dynamic_snitch_badness_threshold:``: The badness threshold will control how much worse the pinned host has to be
+- ``dynamic_snitch_latency_probe_interval_in_ms``: controls how often this node
```

Comment:
Document dynamic snitch class name?

Reply:
OK. I agree we don't want to encourage people to implement their own dynamic snitch.